src/utils/api/api.js

```diff
@@ -58,7 +58,7 @@ async function api(
       options.headers.Accept = 'application/vnd.api+json';
       break;
     case 'content_single':
-      url = `/jsonapi/node/${parameters.bundle}/${parameters.id}`;
```

Comment:
How about not touching the content_single endpoint but keep using the content one instead, given it is now identical?

Reply:
Yes. Good point!